### PR TITLE
Attempt at a Dockerfile

### PR DESCRIPTION
**Pull request by k9lego**
Please do not accept this code yet! Would suggest a review by someone familiar with Dockerfile practices. Otherwise more testing is needed to ensure data loss does not happen regarding persistence between restarts/open channels.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM python:3
```
Comment (k9lego) [inline Dockerfile]: This needs to use a most specific python version

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+ADD . /
```
Comment (k9lego) [inline Dockerfile]: This needs to be adjusted, while it works this isn't ideal

File: Dockerfile (hunk 1 above)
Comment (ChillFish8) [reply Dockerfile]: Bot is currently stable between versions 3.5.3 - 3.7.4 that I've checked, Be aware 3.6.0 has a windows flaw in it and will crash/error out systems using ws and gateways.

Comment (k9lego) [reply Dockerfile]: Sounds good! I'll also look into what version is the most specific version. For NodeJS it's best to use the trimmed-down version of FROM:(Node) in the Dockerfile, I'll try to do the same here.

Comment (ChillFish8) [reply Dockerfile]: Id probably say the best version of python to go with is 3.7.x as has some cleaner functions and is a lil more optimized.
Would be possible if you *really* wanted performance to use my 3.8 PyPy compiler but does require CPP build tools to compile everything for discord.py requirements

**Comment by ChillFish8** [comment]
Docker container was setup on Beta

**Pull request closed without merging**